src/components/transform-object-button.js

```diff
@@ -245,11 +236,13 @@ AFRAME.registerSystem("transform-selected-object", {
     } = this.puppet;
     this.hand.getWorldQuaternion(currentControllerOrientation);
     controllerOrientationDelta.copy(initialControllerOrientation_inverse).premultiply(currentControllerOrientation);
-    this.target.quaternion.copy(initialObjectOrientation).premultiply(controllerOrientationDelta);
+    this.target.quaternion
+      .copy(initialObjectOrientation)
+      .premultiply(controllerOrientationDelta)
+      .premultiply(controllerOrientationDelta);
```

Comment:
Unrelated change (by request) -- doubles the sensitivity of rotation via the puppet-mode rotate button. I agree that this feels.

Reply:
i agree, it does feel.